.eslintrc.json

```diff
@@ -300,7 +300,8 @@
         }
       }
     ],
-    "@typescript-eslint/prefer-nullish-coalescing": ["error"]
+    "@typescript-eslint/prefer-nullish-coalescing": ["error"],
+    "github/array-foreach": ["error"]
```

Comment:
I guess we can replace `github/array-foreach` with [`unicorn/no-array-for-each`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-for-each.md). This gives us one less plugin to install and update. WDYT?